roles/calico/templates/calico.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=calico
+After=docker.service etcd.service
+Requires=docker.service etcd.service
+
+[Service]
+Type=forking
+RemainAfterExit=yes
+Restart=on-failure
+RestartSec=20
+TimeoutStartSec=20m
+
+ExecStartPre=-/usr/bin/docker kill calico-node
+ExecStartPre=-/usr/bin/docker rm calico-node
+Environment="ETCD_AUTHORITY={{ calico_etcdaddr }}"
```

Comment:
we're trying to use `EnvironmentFile` instead of `Environment` for these because it simplifies service reloading. Could you move these into a separate env file please?